defaults/main.yml

```diff
@@ -92,3 +95,34 @@ openwisp2_celery_task_acks_late: true
 openwisp2_celery_broker_max_tries: 10
 openwisp2_django_celery_logging: false
 postfix_smtpd_relay_restrictions_override: "permit_sasl_authenticated, permit_mynetworks, check_relay_domains, reject_unauth_destination, reject"
+openwisp2_users_auth_api: true
+openwisp2_radius_sms_backend: "sendsms.backends.console.SmsBackend"
```

Comment:
we'll need a documentation section in prose mentioning that this is the default value and if user needs to enable this feature they must do X, Y and Z, if you could start a draft regarding this I'll help to fill the blanks.